pyscriptjs/src/pyodide.ts

```diff
@@ -110,6 +110,9 @@ export class PyodideRuntime extends Runtime {
             }
         }
         const response = await fetch(fetch_path);
+        if (response.status !== 200) {
+            throw new Error(`Unable to fetch  ${fetch_path}, reason: ${response.status} - ${response.statusText}`);
+        }
```

Comment:
uhm, maybe we could raise some custom subclass of `Error` here?
This way, we could do a more precise check in `handleFetchError`